### PR TITLE
docformatter: config with black

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes #\<issue_number>

<details>
  <summary>Before submitting</summary>

- [ ] Was this **discussed/agreed** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/torchmetrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

</details>

<details>
  <summary>PR review</summary>

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

</details>

## Did you have fun?

Make sure you had fun coding 🙃

----
:books: Documentation preview :books:: https://torchmetrics--1909.org.readthedocs.build/en/1909/

### CODE REVIEW

**Comment by Borda** [comment]
seems that the 120 lengths is recognized :/

File: pyproject.toml [hunk 1]
```diff
@@ -54,15 +54,11 @@ relative_files = true
...
 
-[tool.isort]
-known_first_party = [
-    "torchmetrics",
-    "integrations",
-    "unittests",
-]
-skip_glob = []
-profile = "black"
-line_length = 120
```
Comment (SkafteNicki) [inline pyproject.toml:65]: Why is this not needed anymore?

Comment (Borda) [reply pyproject.toml:65]: this is not related to this PR so I'll drop it as a special commit but as we merged #1797 we do not use this any more

**Comment by SkafteNicki** [comment]
@Borda pretty sure we need to add
```toml

[tool.docformatter]
recursive = true
wrap-summaries = 120
wrap-descriptions = 120
blank = true

```
to `pyproject.toml` to get it working.

**Comment by Borda** [comment]
> pretty sure we need to add

yes, bad collision resolving, sorry

**Pull request merged**